overview.dd

```diff
@@ -1044,23 +1067,25 @@ $(SECTION2 Sample D Program (sieve.d),
 
 import std.stdio;
 
-bool[8191] flags;
-
-int main()
+void main()
 {
-    int i, count, prime, k, iter;
+    size_t i, count, prime, k;
+    bool[8191] flags;
 
-    writeln("10 iterations");
-    for (iter = 1; iter <= 10; iter++)
+    "10 iterations".writeln;
+    
+    // using _ as a throwaway variable
+    foreach(_; 1 .. 11)
```

Comment:
Please no, just use `iter`.

`_` is not what people might expect, if they come from languages where it is an actual language syntactical feature. You can't define more than one `_` in a function in D, for example, because it's just a variable.